diktat-rules/pom.xml

```diff
@@ -168,7 +168,7 @@
                         <configuration>
                             <artifacts>
                                 <artifact>
-                                    <file>${build.directory}/diktat.jar</file>
+                                    <file>${build.directory}/diktat-${project.version}-jar-with-dependencies.jar</file>
```

Comment:
we really need such long name?

Reply:
I believe it's standard classifier for fat jars. If we want to deploy fat jat simply as `diktat-${version}.jar`, I can add this configuration as a separate maven module.

Reply:
standard pattern is fat-jar-with-dependencies, but I guess `diktat-$version-fat.jar` is absolutely fine and more usable